org.kde.kstars.json

```diff
@@ -1,33 +1,31 @@
 {
     "id": "org.kde.kstars",
     "runtime": "org.kde.Platform",
-    "runtime-version": "5.15-24.08",
+    "runtime-version": "6.9",
     "sdk": "org.kde.Sdk",
     "command": "kstars",
     "rename-icon": "kstars",
+    "tags": ["stable", "3.7.7"],
     "finish-args": [
         "--share=ipc",
         "--socket=fallback-x11",
         "--socket=wayland",
-        "--device=dri",
-        "--share=network"
-    ],
-    "cleanup": [
-        "/include",
-        "/lib/pkgconfig",
-        "/lib/*.a",
-        "/lib/*.la",
-        "/share/man"
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--system-talk-name=org.bluez",
+        "--talk-name=org.freedesktop.Notifications",
```

Comment:
Qt should be using the portal for that. why is it needed?

Reply:
no need